docs/WifiPerformance.md

```
@@ -324,1071 +324,58 @@ This section presents the performance test results, including key metrics and te
-    tx bitrate: 867.0 MBit/s
-    ```
-
-#### Realtek 8821CU #1
-
-<img src=https://stuff.armbian.com/netbox/media/devicetype-images/RTL8821CU.png>
-<span style="font-size: 0.5rem;">OS: Armbian v25.5.0-trunk.550, 6.12.28-current-x86</span>
-
-| Chipset | Class | Average forward speed | Average reverse speed |
-|:-----|------|-------:|-------:|
-|<span style="font-size: 1.5rem;">RTL8821CU</span> | <span style="font-size: 1.5rem;">AC</span> | <span style="font-size: 1.5rem;">123</span> Mbits/sec | <span style="font-size: 1.5rem;">211</span> Mbits/sec |
+|<span style="font-size: 1.5rem;">RTL8812AU</span> | <span style="font-size: 1.5rem;">AC</span> | <span style="font-size: 1.5rem;">80.8</span> Mbits/sec | <span style="font-size: 1.5rem;">71.5</span> Mbits/sec |
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: span

(MD033, no-inline-html)